src/textbracket.ts

```diff
@@ -70,30 +70,19 @@ export class TextBracket extends Element {
     };
   }
 
-  /**
-   * @deprecated Use `TextBracket.Position` instead.
-   */
-  static get Positions(): typeof TextBracketPosition {
-    L('Positions is deprecated, use TextBracketPosition instead.');
-    return TextBracketPosition;
-  }
-
-  /**
-   * @deprecated Use `TextBracket.PositionString` instead.
-   */
-  static get PositionsString(): Record<string, number> {
-    L('PositionsString is deprecated, use PositionString instead.');
-    return TextBracket.PositionString;
-  }
-
   constructor({ start, stop, text = '', superscript = '', position = TextBracketPosition.TOP }: TextBracketParams) {
     super();
 
     this.start = start;
     this.stop = stop;
 
-    this.text = text;
-    this.superscript = superscript;
+    this.textElement = new Element('TextBracket');
+    this.textElement.setText(text);
+    this.textElement.measureText();
+    this.superscriptElement = new Element('TextBracket');
+    this.superscriptElement.setText(superscript);
+    const smallerFontSize = Font.scaleSize(this.textFont.size, 0.714286);
```

Comment:
Just curious where the 0.714... comes from? :-)

Reply:
It was already present in the original source. I agree, that it is not nice but I wanted to minimize the differences.